sdk/docs/sharable/sdk/quickstart/configure/quickstart-project-structure/04-quickstart-top-level-dir.rst

```diff
@@ -66,43 +65,38 @@ incorporated into the build process in `daml/build.gradle.kts`.
    * - `VersionFiles.kt`
      - Provides access to `.env` files and `daml.yaml` files from Gradle.
 
-The project also uses Make [13]_ as a project choreographer, providing a
-convenient command-line interface to the various scripts and build tools
-as well as docker-compose commands. This is similar to the common
-practice of defining aliases for common dev-loop tasks. Make has the
-advantage of documenting and sharing these tasks under revision
-control. [14]_ Use `make help` to view the currently supported tasks. The
-`Makefile` itself is intended to be implicit documentation of how each of
-these steps is performed. By default, `make` also prints any commands it
-executes to `stdout` and this can also help familiarize new developers with
-how the dev-loop is structured. If your team is unfamiliar with Make, at
-the end of this guide [15]_, we have documented the Make features used
-in the current Makefile with links to additional documentation.
+The project also uses Make [12]_ as a project choreographer, providing a convenient command-line interface to the various scripts and build tools as well as docker-compose commands. 
+This is similar to the common practice of defining aliases for common dev-loop tasks.
+Make has the advantage of documenting and sharing these tasks under revision control. [13]_ 
+Use ``make help`` to view the currently supported tasks.
+The ``Makefile`` itself is intended to be implicit documentation of how each of these steps is performed. 
+By default, ``make`` also prints any commands it executes to ``stdout`` and this can also help familiarize new developers with how the dev-loop is structured. 
+If your team is unfamiliar with Make, at the end of this guide [14]_, 
+we have documented the Make features used in the current Makefile with links to additional documentation.
+
+Deployment configuration
+------------------------
 
-Local deployment (LocalNet) configuration
------------------------------------------
+Local deployment is handled via Docker [15]_ and Docker Compose [16]_ in the usual fashion. 
+Like other blockchains, it constructs a `LocalNet` on your laptop. 
+In summary:
 
-Local deployment is handled via Docker [16]_ and Docker Compose [17]_ in
-the usual fashion. Like other blockchains, it constructs a `LocalNet` on
-your laptop. In summary:
+``.env`` and ``.env.local`` define the necessary environment variables.
```

Comment:
Not quite true.  With the new setup, each docker image has its env vars colocated.